deployment/h2o/h2o-deployment.ipynb

```diff
@@ -22,7 +22,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```

Comment:
My commit added these because I have a notebook hook that numbers the cells and I forget how to turn it off 🤷